packages/cdk/cdk.tenant.example.json

```
@@ -1,11 +1,8 @@
 {
   "context": {
     "tenantId": "tenant123",
-    "identityProviderArn": "arn:aws:cognito-idp:us-east-1:123456789012:userpool/us-east-1_XXXXXXXX",
-    "audience": "your-client-id",
-    "tenantIdClaim": "custom:tenant_id",
+    "environment": "dev",
```

cdk.jsonの方と合わせた命名の方が望ましい
```suggestion
    "env": "dev",
```